daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/transaction/TransactionVersion.scala

```diff
@@ -23,7 +23,7 @@ object TransactionVersion {
 
   val All = List(V10, V11, V12, V13, V14, VDev)
 
-  private[daml] implicit val Ordering: scala.Ordering[TransactionVersion] =
+  private implicit val Ordering: scala.Ordering[TransactionVersion] =
```

Comment:
Otherwise it will be private to `TransactionVersion`
```suggestion
  implicit val Ordering: scala.Ordering[TransactionVersion] =
```